.github/workflows/tests.yml

```diff
@@ -231,7 +231,7 @@ jobs:
   core_frontend_tests:
     if: "!(github.event_name == 'push' && github.repository == 'discourse/discourse-private-mirror')"
     name: core frontend (${{ matrix.browser }})
-    runs-on: ubuntu-20.04-8core
+    runs-on: buildjet-4vcpu-ubuntu-2204
```

Comment:
Just curious, is there a reason we're dropping this down from 8 core to 4 cores?

Reply:
Yeah.. there's a need to fit all runs within 64 cores. Time-wise we're only seeing a 1minute ish difference.
 
[BuildJet](https://github.com/discourse/discourse/actions/runs/4911344097/usage)
<img width="484" alt="Screenshot 2023-05-09 at 9 39 13 AM" src="https://user-images.githubusercontent.com/1555215/236972425-81bf5346-c66b-4303-b2cf-eba465c7f4e3.png">

[Github](https://github.com/discourse/discourse/actions/runs/4920835628/usage)
<img width="484" alt="Screenshot 2023-05-09 at 9 39 36 AM" src="https://user-images.githubusercontent.com/1555215/236972470-1564d947-f94f-4bba-b181-5f4f5f8e4f5f.png">